ext/moderation.py

```diff
@@ -0,0 +1,185 @@
+import disnake
+from disnake.ext import commands
+
+from utils.bot import Cog
+from utils.checks import convert
+
+
+class moderation(Cog):
+    @commands.slash_command(name="ban")
+    @commands.has_permissions(ban_members=True)
+    async def ban(
+        self, inter: disnake.ApplicationCommandInteraction, member: disnake.Member, reason: str = "No reason"
+    ):
+        """Bans a member
+
+        Parameters
+        -----------
+        member: The member to ban
+        reason: The reason for banning
+        """
+        await inter.send(
+            embed=disnake.Embed(
+                title="Member banned",
+                description=f"⚒ | {member.mention} has been wiped out from this server by {inter.user.mention} for **{reason}**!",
+                color=disnake.Color.red(),
+            )
+        )
+        await member.send(
+            embed=disnake.Embed(
+                title="Banned",
+                description=f"You have been banned from {inter.guild.name} by {inter.user.mention} for **{reason}**!",
+            )
+        )
+        await inter.guild.ban(member)
+
+    @commands.slash_command(name="tempban")
+    async def tempban(
+        self,
+        inter: disnake.ApplicationCommandInteraction,
+        member: disnake.Member,
+        duration: str,
+        reason: str = "No reason",
+    ):
+        """Temporary bans a member
+
+        Parameters
+        ----------
+        member: The member to temp ban
+        duration: The duration for banning. (s, m, h ,d format)
+        reason: The reason for temp banning
+        """
+        duration = round(disnake.utils.utcnow().timestamp() + convert(duration))
+
+        await inter.send(
+            embed=disnake.Embed(
+                title="Member banned",
+                description=f"⚒ | {member.mention} has been wiped out from this server by {inter.user.mention} for **{reason}**!",
+                color=disnake.Color.red(),
+            )
+        )
+
+        await inter.guild.ban(member)
+        await self.bot.db.execute(
+            f"INSERT INTO tempbans (guild_id, target_id, unban_time) VALUES ({inter.guild.id}, {member.id}, {duration})"
+        )
```

Comment:
No f-strings, use placeholders

Reply:
using place holders giving me error that's why used it ;-;